### DISCUSSION TITLE
Operator give a visibility on how much co2 is saved ?

### DISCUSSION THREAD

**Discussion by foxutech** [kube-green/kube-green#320] [Ideas]
Hi Team, 

 As currently, there are multiple organizations exposing options to reduce CO2, is the kube-green can provide like visibility on how much CO2 it saved will be interesting metric. 

Thanks, 
FoxuTech

**Comment by davidebianchi** [comment]
Hi! Do you mean something like a dashboard to see how many CO2 are saved in a specific period?

**Reply by foxutech** [reply]
Yes either a metrics dashboard or something to expose with Prometheus.  Instead of own kube-green dashboard(until there is quite number of options), prometheus metrics would be best.
